jdk/test/java/util/Currency/ValidateISO4217.java

```diff
@@ -23,8 +23,8 @@
 /*
  * @test
  * @bug 4691089 4819436 4942982 5104960 6544471 6627549 7066203 7195759
- *      8074350 8074351 8145952 8187946 8193552 8202026 8204269
- *      8208746 8209775 8274658 8283277 8296239
+ *      8039317 8074350 8074351 8145952 8187946 8193552 8202026 8204269
+ *      8208746 8209775 8264792 8274658 8283277 8296239 8321480
```

Comment:
This should only be a one-line change appending `8321480` to the last line. See the [JDK 11u commit](https://github.com/openjdk/jdk11u-dev/commit/ea06b5c966e9261cacb35300ddde6eae48ec632f#diff-07e0488dd131c8c4cfcf7c135041095716c42fdeacaf67e8f37475bf9c4780ffR27) which does the same thing.

Reply:
@jerboaa Ack.  bc18ee3